lib/JMAP/TestSuite/ServerAdapter/Cyrus.pm

```diff
@@ -11,6 +11,10 @@ has base_uri => (
   required => 1,
 );
 
+has '+can_use_websockets' => (
```

Comment:
This is fine here, but I think we should probably avoid using `has +foo` and use builders more often.  Otherwise, we eventually get bitten by the things that `has +foo` can't do.